tests/create_grid_2d_test.py

```diff
@@ -62,3 +80,16 @@ def test_create_grid_2d_data(tmp_path):
 
     compare_entities(grid, rec_obj)
     compare_entities(data, rec_data)
+
+    with pytest.raises(IndexError, match="Only 1, 3, or 4 layers can be selected"):
+        grid.to_geoimage(["DataValues", "DataValues"])
+
+    geoimage = grid.to_geoimage(["DataValues"])
+
+    assert isinstance(geoimage, GeoImage)
+
+    geoimage = grid.to_geoimage(["DataValues", "DataValues", "DataValues"])
```

Comment:
Why is it still showing as black and white after?

![image](https://user-images.githubusercontent.com/55204635/204352741-12708529-573f-4f79-a16d-60691a82e474.png)

Reply:
If each channel are the same, the color is a grayscale.